dicom-mr-classifier.py

```diff
@@ -583,8 +583,8 @@ def dicom_classify(zip_file_path, outbase, timezone, config=None):
             classification = classification_from_label.infer_classification(series_desc)
             log.info("Inferred classification from label: %s", classification)
             # GEAR-1084, keep any custom classification already set.
-            if not classification:
-                classification = {'Custom': ['N/A']}
+        if not classification:
+            classification = {'Custom': ['N/A']}
```

Comment:
So right now, 
if series_desc, that's returning `{}`, if not and no classification `{'Custom': ['N/A']}`. Kind of odd.

What about something like to streamline it:
```
    classification = {}
    if series_desc:
        metadata["acquisition"]["label"] = series_desc
        classification = get_custom_classification(series_desc, config)
        log.info("Custom classification from config: %s", classification)
        if not classification and dcm.get("Modality") == "MR":
            classification = classification_from_label.infer_classification(series_desc)
            log.info("Inferred classification from label: %s", classification)

    if not classification:
        classification = {'Custom': ['N/A']}
    dicom_file["classification"] = classification
```

Reply:
And updating `get_custom_classification` to make it return {} to preserve typing somehow.

Reply:
Will this simpler fix work?